### PR TITLE
core: install storybook

### PR DESCRIPTION
**Pull request by ZadorozhnyiYevhenii**

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -3,12 +3,7 @@ require('@rushstack/eslint-patch/modern-module-resolution')
 
 module.exports = {
   root: true,
-  'extends': [
-    'plugin:vue/vue3-essential',
-    'eslint:recommended',
-    '@vue/eslint-config-typescript',
-    '@vue/eslint-config-prettier/skip-formatting'
-  ],
+  'extends': ['plugin:vue/vue3-essential', 'eslint:recommended', '@vue/eslint-config-typescript', '@vue/eslint-config-prettier/skip-formatting', 'plugin:storybook/recommended'],
```
Comment (P0TER) [inline .eslintrc.cjs:6]: Нам надо настроить форматирование)

Comment (P0TER) [inline .eslintrc.cjs:6]:
```suggestion
  'extends': [
    'plugin:vue/vue3-essential',
    'eslint:recommended',
    '@vue/eslint-config-typescript',
    '@vue/eslint-config-prettier/skip-formatting',
    'plugin:storybook/recommended'
  ],
```

Comment (ZadorozhnyiYevhenii) [reply .eslintrc.cjs]: Зробимо)

**Review by P0TER — APPROVED** [review]

**Comment by P0TER** [comment]
This PR is included in version 1.0.0 :tada:

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0 :tada:

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0 :tada:

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0 :tada:

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0 :tada:

**Comment by P0TER** [comment]
This pull request is included in version 1.0.0 :tada:

**Pull request merged**